src/staging/staging_main.py

```diff
@@ -245,6 +245,13 @@ def run_staging(
         val.validate_data_with_schema,
         val.validate_ultfoc_df,
     )
+    # # Debug begin
+    # cora_mapper = cora_mapper.astype("str")
+    # StagingMainLogger.info(f"Cora mapper coulms:\n{cora_mapper.dtypes}")
+    # # Debug end
+
+    # cora_mapper = val.validate_cora_df(cora_mapper)
+    # StagingMainLogger.info("Cora status mapper file loaded successfully...")
```

Comment:
Assume this needs to be taken out (all refs to cora mapper will taken out with 574)?